src/airtable.js

```diff
@@ -51,6 +52,27 @@ exports.findRequestByExternalId = async externalId => {
   }
 };
 
+exports.findOpenRequests = async () => {
+  const requestOpenStates = ["Dispatch Started", "Delivery Needed"];
+  const statusConstraints = requestOpenStates.map(s => `{Status} = '${s}'`);
+  const formula = `OR(${statusConstraints.join(", ")})`;
+  console.log(`Finding: ${formula}`);
+  try {
+    const requests = await base("Requests")
+      .select({
+        filterByFormula: formula
+      })
+      .all();
+    const notInSlack = r => {
+      const meta = JSON.parse(r.get("Meta"));
+      return meta.slack_ts === undefined;
```

Comment:
I think we need to leave the ability to post even if it already exists in slack. The state should be the only indicator. The reason being is that it's pretty common to cross post between channels

I think storing this in the meta field would be useful for the polling logic below, but:
1. might be better suited for a separate PR
2. Going with opt-in approach, would be better handled by the "Assign to Delivery" shortcut

I think this would remove the need for the slack channel + ts in the meta field. 
I know this is a big part of this PR. What are your thoughts?

Reply:
Shouldn't cross-posts be done by linking the post rather than duplicating it? Otherwise the threads get out of control. 
![image](https://user-images.githubusercontent.com/57376/78623142-00407900-7855-11ea-9b35-8e91699be186.png)

Regardless, yeah we can definitely make all of the updating contingent on actions in slack to start with. If it ends up that volunteers end up starting a message through this and then finishing up in airtable for some reason we can automate it.

Reply:
Though we would need the ability to mark a request as complete too to launch (because these can't be edited). Ideally I think there would be an action to do a raw edit as well for cases where the workflow actions we have don't match with reality?

Reply:
> Shouldn't cross-posts be done by linking the post rather than duplicating it?

Yes, but I'm not sure how many volunteers know how to do this. I guess worst case they can copy paste, but I'd prefer to get people used to using the shortcut

> If it ends up that volunteers end up starting a message through this and then finishing up in airtable for some reason we can automate it. Though we would need the ability to mark a request as complete too to launch (because these can't be edited).

The "completed" is actually "assigned" which can be triggered through the "Assign to Delivery shortcut" (not currently implemented)

> Ideally I think there would be an action to do a raw edit as well for cases where the workflow actions we have don't match with reality?

Definitely agree, but not a blocker for this